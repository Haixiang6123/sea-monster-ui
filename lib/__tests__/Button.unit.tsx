import * as React from 'react'
import renderer from 'react-test-renderer'
import Button from '../Button'

describe('Button', () => {
  it('is a div', () => {
    const tree = renderer.create(<Button/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
