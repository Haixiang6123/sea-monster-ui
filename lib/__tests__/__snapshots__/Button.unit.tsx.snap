// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button is a div 1`] = `
<div>
  button
</div>
`;
